servicetalk-examples/grpc/debugging/src/main/java/io/servicetalk/examples/grpc/debugging/DebuggingClient.java

```diff
@@ -149,7 +150,7 @@ public static void main(String[] args) throws Exception {
                          * Be sure to also enable the logger in your logging config file,
                          * {@code log4j2.xml} for this example.
                          */
-                        .protocols(HttpProtocolConfigs.h2()
+                        .protocols(h1Default(), HttpProtocolConfigs.h2()
```

Comment:
This will result in a runtime exception. 2 protocols are supported only when there is SSL configuration provided, bcz it requires ALPN.

Reply:
OK, any suggestion what we should do with this example? It is misleading in the current form in that it demonstrates something that is disabled.

Reply:
This is gRPC example, it's expected to work only over HTTP/2. Can you clarify what exactly is misleading?